region/norfair/crocomire/Indiana Jones Room.json

```diff
@@ -711,6 +711,279 @@
+        "canTrickyJump"
+      ],
+      "exitCondition": {
+        "leaveWithMockball": {
+          "remoteRunway": {
+            "length": 32,
+            "openEnd": 1,
+            "gentleUpTiles": 6
+          },
+          "landingRunway": {
+            "length": 10,
+            "openEnd": 1
+          },
+          "minExtraRunSpeed": "$5.9"
+        }
+      },
+      "unlocksDoors": [{"types": ["ammo"], "requires": [], "useImplicitRequires": false}],
+      "flashSuitChecked": true
+    },
+    {
+      "link": [2, 1],
+      "name": "Leave With Mockball (Long Runway)",
+      "requires": [
+        {"obstaclesCleared": ["A", "B", "E"]},
+        "canSpeedyJump",
+        "canTrickyJump"
+      ],
+      "exitCondition": {
+        "leaveWithMockball": {
+          "remoteRunway": {
+            "length": 45,
+            "openEnd": 1
+          },
+          "landingRunway": {
+            "length": 10,
+            "openEnd": 1
+          },
+          "minExtraRunSpeed": "$6.D"
+        }
+      },
+      "unlocksDoors": [{"types": ["ammo"], "requires": [], "useImplicitRequires": false}],
+      "flashSuitChecked": true
+    },
+    {
+      "link": [2, 1],
+      "name": "Leave With Spring Ball Bounce (Short Runway, Space Jump)",
+      "requires": [
+        {"obstaclesCleared": ["B", "E"]},
+        "canPreciseSpaceJump",
+        "canInsaneJump"
+      ],
+      "exitCondition": {
+        "leaveWithSpringBallBounce": {
+          "remoteRunway": {
+            "length": 32,
+            "openEnd": 1,
+            "gentleUpTiles": 6
+          },
+          "landingRunway": {
+            "length": 10,
+            "openEnd": 1
+          },
+          "movementType": "uncontrolled"
+        }
+      },
+      "unlocksDoors": [{"types": ["ammo"], "requires": [], "useImplicitRequires": false}],
+      "flashSuitChecked": true
+    },
+    {
+      "link": [2, 1],
+      "name": "Leave With Spring Ball Bounce (Long Runway, Space Jump)",
+      "requires": [
+        {"obstaclesCleared": ["A", "B", "E"]},
+        "canPreciseSpaceJump",
+        "canInsaneJump"
+      ],
+      "exitCondition": {
+        "leaveWithSpringBallBounce": {
+          "remoteRunway": {
+            "length": 45,
+            "openEnd": 1
+          },
+          "landingRunway": {
+            "length": 10,
+            "openEnd": 1
+          },
+          "movementType": "uncontrolled"
+        }
+      },
+      "unlocksDoors": [{"types": ["ammo"], "requires": [], "useImplicitRequires": false}],
+      "flashSuitChecked": true
+    },
+    {
+      "link": [2, 1],
+      "name": "Leave With Controlled Spring Ball Bounce (Short Runway, Mid-Air Jump)",
+      "requires": [
+        {"obstaclesCleared": ["B", "E"]},
+        "canTrickySpringBallJump",
+        "canTrickySpringBallBounce",
+        "canInsaneJump"
+      ],
+      "exitCondition": {
+        "leaveWithSpringBallBounce": {
+          "remoteRunway": {
+            "length": 32,
```

Comment:
Could this say you don't use the full runway (do not have speed) and can reach the door (do not have walljump)?